src/test/scala/io/citrine/lolo/bags/BaggerTest.scala

```diff
@@ -77,6 +77,35 @@ class BaggerTest {
   }
 
   /**
+    * Test that the uncertainty metrics are properly calibrated
+    *
+    * This test is based on the "standard" RMSE, which is computed by dividing the error
+    * by the predicted uncertainty.  On the exterior, these is an additional extrapolative
+    * uncertainty that isn't captured well by this method, so we test the interior and the full
+    * set independently
+    */
+  def testUncertaintyCalibration(): Unit = {
+    val width = 0.1
+    val trainingData = TestUtils.generateTrainingData(128, 6, xscale = width, seed = 6L)
+    val DTLearner = new RegressionTreeLearner(numFeatures = 6)
+    val bias = new RegressionTreeLearner(maxDepth = 4)
+    val baggedLearner = new Bagger(DTLearner, numBags = 16 * trainingData.size, biasLearner = Some(bias))
```

Comment:
Why 16?

Reply:
Large to reduce the influence of the bias correction (which causes over-estimation).  16 speciifcally because it was the smallest value that gave good agreement with 1.0

Reply:
I'd pull this magic number out into its own variable, and include a comment to the effect of what you've said above.